community/examples/storage-gke.yaml

```diff
@@ -76,3 +79,57 @@ deployment_groups:
       - echo \$(date) >> /shared/timestamp.log; cat /shared/timestamp.log
       node_count: 3
     outputs: [instructions]
+
```

Comment:
I agree that trying to enforce this can easily do more harm than good and should rely on K8S for user actions.